ecommerce-ui/src/RentalCart/RentalCart.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import RentalCart from './RentalCart';
+import TestRenderer from 'react-test-renderer';
```

Comment:
When installing third-party modules like `react-test-renderer`, make sure to use the `--save` flag so it's saved to package.json